src/types/Calculator.ts

```diff
@@ -97,11 +97,27 @@ export interface ExecutionResult<T extends PopulationGroupResult> {
  */
 export interface SDEResult {
   /** Name of the SDE */
-  name: string;
+  name?: string;
   /** Raw result of SDE clause */
   rawResult?: any;
   /** Pretty result for this SDE. */
   pretty?: string;
+  /** The id of the SDE as defined in the measure. */
+  id?: string;
+  /** Criteria expression for this SDE. */
+  criteriaExpression?: string;
+  /** Measure data usage for this SDE. */
+  usage: SupplementalDataUsage;
+}
+/**
```

Comment:
Nitpicking, but maybe a newline here for readability :)

Reply:
I guess we don't really do that anywhere else. I'll just trust people to shift their eyes one more line if needed lol